### PR TITLE
Add Tip to using  MailHog during developement

### PR DESCRIPTION
**Pull request by zairigimad**
Add Tip to using  MailHog during developement

### CODE REVIEW

File: email/dev_environment.rst [hunk 1]
```diff
@@ -13,6 +13,11 @@ choices when it comes to handling email during development: (a) disabling the
 sending of email altogether or (b) sending all email to a specific
 address (with optional exceptions).
 
+.. tip::
+
+    for testing emails during application development, and to make tests simple
```
Comment (OskarStark) [inline email/dev_environment.rst]:
```diff
- for testing emails during application development, and to make tests simple
- to web designers, you can use  `MailHog`_. 
+ For testing emails during development you can use `MailHog`_.
```

**Comment by javiereguiluz** [comment]
I've never used this tool before. Before merging this, we'd need to hear opinions about this tool from Symfony developers who have used it. Thanks!

**Comment by kiler129** [comment]
What's is the advantage over https://mailcatcher.me recommended by SwiftMailer docs?

**Comment by jewome62** [comment]
In all the projects I could see, when the mail was not disabled, it was mailcatcher that was used.
With most of the time, use the lib to do behat tests with: https://github.com/alexandresalome/mailcatcher
since most of the time we use docker, this is an image to use simply

**Comment by artemzi** [comment]
@zairigimad are sure that project is live? the latest commit was a year ago and there is a lot of issues there. I prefer to use mailtrap.io

**Comment by Pierstoval** [comment]
Mailcatcher is really easy to set up, especially in docker (just create a compose service based on mailcatcher & share networks and that's all), what are the benefits of MailHog? What's the difference?

**Review by HeahDude — APPROVED** [review]
Thanks for this tip! We indeed use MailCatcher in many projects.

**Comment by javiereguiluz** [comment]
This is now merged. Thank you all for your comments!

**Pull request closed without merging**